android/app/src/main/java/co/nimblehq/smsforwarder/ui/base/NavigationEvent.kt

```diff
@@ -4,6 +4,7 @@ import co.nimblehq.smsforwarder.ui.screens.second.SecondBundle
 import co.nimblehq.smsforwarder.ui.screens.webview.WebViewBundle
 
 sealed class NavigationEvent {
+    object Filter : NavigationEvent()
```

Comment:
🤔  Why don't we define the `Home` event here rather than `Filter`? Once user logged-in successfully, the app will navigate to the Home screen 🤔

Reply:
Oh, True, I thought the name was swapped. 😄  Will update.